server/models/HistoryEra.js

```diff
@@ -45,6 +47,18 @@ class HistoryEra {
     this.startAutoCleanup(cleanupInterval, this.db);
   }
 
+  loadDatabase(userId, dbName) {
```

Comment:
As I commented on `ClientRequest`'s `checkHash` method , let's retrieve `userId` on the instance rather than this extra argument.

Reply:
agree